cmd/api/main.go

```diff
@@ -446,6 +424,15 @@ func createChainStacks(
 		chainStacks[chainCfg.ChainID] = chainStack
 	}
 
+	epMap := make(map[tableland.ChainID]eventprocessor.EventProcessor)
```

Comment:
This logic is kind of ugly, but I could not think of something better. In the user store [we needed to pass the resolver](https://github.com/tablelandnetwork/go-tableland/pull/495/files#diff-ae22ad1e20b89eaefa8e09e4ed89443ac10fdc555b9767625f3866e3853100afL102). Since we transferred that functionality to the system store, we gotta do the same to the system store. But we have one system store per chain, that's why we got to do these loops, and set the resolver for each system store.